examples/next-login/src/pages/home/signin.tsx

```diff
@@ -0,0 +1,207 @@
+import { Text, Stack, Alert } from '@ultraviolet/ui'
+import { Icon } from '@ultraviolet/icons'
+import {
+  Form,
+  TextInputFieldV2,
+  TextInputField,
+  Submit,
+  useForm,
+  RadioField,
+  DateField,
+  FormErrors,
+} from '@ultraviolet/form'
+import { Theme, css, useTheme } from '@emotion/react'
+import { useState } from 'react'
+
+type FormValues = {
+  email: string
+  password: string
+  gender: string
+  firstname: string
+  lastname: string
+  birthdate: Date
+}
+
+const emailRegex = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i
+const mockErrors: FormErrors = {
```

Comment:
Instead of declaring it in both files (login, register) you could put this into a `helper.ts` file and import it in both files